scout/log/log.py

```diff
@@ -0,0 +1,63 @@
+import os
+import sys
+import logging
+
+def init_log(logger, filename=None, loglevel=None):
```

Comment:
Sure thing - I think we can update the formatted template line first though :smiley: